torchvision/models/video/resnet.py

```diff
@@ -189,9 +211,15 @@ def __init__(self):
 
 class VideoResNet(nn.Module):
 
-    def __init__(self, block, conv_makers, layers,
-                 stem, num_classes=400,
-                 zero_init_residual=False):
+    def __init__(
+        self,
+        block: Type[Union[BasicBlock, Bottleneck]],
+        conv_makers: List[Type[Union[Conv3DSimple, Conv3DNoTemporal]]],
+        layers: List[int],
+        stem: nn.Module,
```

Comment:
This should be optional as per the documentation.

Reply:
The documentation doesn't seem correct then, or is it?
Otherwise I need to specify a default value, should I put `BasicStem`?

Reply:
Let's fix the documentation instead